tutorials/financial-tick-data/ingest-real-time-websocket-data.md

```diff
@@ -0,0 +1,438 @@
+---
+title: Ingest real-time financial websocket data
+excerpt: Set up a data pipeline to get data from different financial APIs
+products: [cloud, mst, self_hosted]
+keywords: [finance, analytics, websockets, data pipeline]
+---
+
+import CreateHypertableStocks from "versionContent/_partials/_create-hypertable-twelvedata-stocks.mdx";
+import GraphOhlcv from "versionContent/_partials/_graphing-ohlcv-data.mdx";
+
+# Ingest real-time financial websocket data
+
+This tutorial shows you how to ingest real-time time-series data into
+TimescaleDB using a websocket connection. The tutorial sets up a data pipeline to
```

Comment:
```suggestion
In this tutorial you will learn how to ingest real-time time-series data into
TimescaleDB using a websocket connection. The tutorial sets up a data pipeline to
```

Reply:
https://docs.timescale.com/about/latest/contribute-to-docs/#w